components/taggroup/stories/taggroup.stories.js

```diff
@@ -99,15 +101,19 @@ OverflowItems.args = {
 		},
 		{
 			label: "Tag 4",
+			avatarUrl: "example-ava.png",
 		},
 		{
 			label: "Tag 5",
+			isInvalid: true,
 		},
 		{
 			label: "Tag 6",
+			isEmphasized: true,
 		},
 		{
 			label: "Tag 7",
+			isDisabled: true,
```

Comment:
We weren't showing a lot of different tag states/variants, which is fine because that should be Tag's responsibility, but I didn't think there was much harm in showing a few different variants, happy to change it back if there's a good rationale for _not_ showing different tag variants.

Reply:
I have no problem with that! I'll take out the disabled/emphasized/invalid variants and keep the avatar tag.